dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -69,7 +74,12 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             {
                 THROW_IF_FAILED(PushNotifications_RegisterFullTrustApplication(toastAppId.c_str(), GUID_NULL));
 
-                storedComActivatorString = RegisterComActivatorGuidAndAssets(details);
+
+                // TODO: Add permanent displayname and iconuri.
+                wil::unique_cotaskmem_string processName;
```

Comment:
Put this outside the if-condition. You are reunsing it below too and the calls are redundant.